shared/js/background/storage/tds.es6.js

```diff
@@ -28,6 +28,10 @@ class TDSStorage {
                     },
                     exceptions: []
                 },
+                trackingCookies3p: {
+                    state: 'enabled',
+                    exceptions: []
```

Comment:
Sorry I don't understand? There is now a separate trackingCookies1p that may have its own exceptions list to act on the first party.

Reply:
For all features in the config, if `example.com` is in `exceptions` it means: this feature is disabled when the site url is `example.com`.

However, `trackerCookies3p`, works differently. In this case on _any_ site, 3rd party cookies to `example.com`. I'm saying that
 1. Because the semantics of this exception differ to how `exceptions` are handled in all other cases, the name of this list in the config should be different to prevent confusion. e.g. `3pCookiesAllowed`.
 2. Perhaps we should also support the same kind of site exception for this feature too (i.e. disabling 3p cookie blocking on `example.com`) under the `exceptions` key for consistency.

Reply:
But we ARE disabling 3p cookie blocking on sites in this exceptions list. Changing the name of this list would be highly inconsistent in the config and cause maintainability headaches.